openlibrary/components/IdentifiersInput.vue

```diff
@@ -4,12 +4,12 @@
       <th>
         <select class="form-control" v-model="selectedIdentifier" name="name">
           <option disabled value="">Select one</option>
-          <template v-if="isEdition">
-            <option v-for="entry in popularEditionConfigs" :key="entry.name" :value="entry.name">
+          <template v-if="Object.keys(popularIdDict).length !== 0">
```

Comment:
This might be better as a computed, since checking for an empty dict in JS is a bit of a mouthful! 
```suggestion
          <template v-if="hasPopularIds">
```